FrEIA/modules/base.py

```diff
@@ -0,0 +1,38 @@
+from typing import Tuple, Iterable, Union
+
+import torch.nn as nn
+from torch import Tensor, is_tensor
+
+
+class InvertibleOperator(nn.Module):
+    def __init__(self, dims_in: Tuple[int], dims_c: Tuple[int] = None):
+        super().__init__()
+        if dims_c is None:
+            dims_c = []
+        self.dims_in = dims_in
+        self.dims_c = dims_c
+
+    def forward(self, x: Union[Iterable[Tensor], Tensor], c: Union[Iterable[Tensor], Tensor], rev=False, jac=True):
+        # Tupelize input
```

Comment:
```suggestion
        """
        perform a forward pass through this module/operator
        
        input parameters:
        x   .. input data (can be an iterable over a tensor or a tensor)
        c   .. conditioing data (can be an iterable over a tensor or a tensor)
        rev .. perform backward pass
        jac .. return jacobian
        """
        # Tupelize input
```

As I don't see a docstring standard for freia. I made up one - which is bad in itself.